02Lesson/01Homework/index.html

```diff
@@ -1,8 +1,54 @@
 <!DOCTYPE html>
-<html>
-<head>
-  <title></title>
-</head>
-<body>
-</body>
+<html style= "background: lightblue" >
```

Comment:
No space after `=` sign. Missing semicolon. Styles are generally not applied to the `html` element, use body element instead, or even better, classes etc.